tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -76,6 +76,18 @@ stages:
       artifactName: 'onnxruntime-android-full-aar'
       job_name_suffix: 'Full'
 
+- stage: Android_Java_API_AAR_Packaging_QNN
+  dependsOn: []
+  jobs:
+  - template: android-qnn-java-api-aar.yml
+    parameters:
+      buildConfig: 'Release'
+      buildSettings: '$(Build.SourcesDirectory)/tools/ci_build/github/android/default_qnn_aar_build_settings.json'
+      artifactName: 'onnxruntime-android-qnn-aar'
+      job_name_suffix: 'QNN'
+      publish_executables: '0'
+  #TODO: Add test job for QNN Android AAR
```

Comment:
Now that the initial browerstack setup for Android is checked in we could try and do this (separate PR though). 

More important to get the initial pre-release package published and validated though so manual testing is fine short term.